projects/storefrontstyles/scss/cxbase/blocks/alert.scss

```diff
@@ -4,70 +4,70 @@
 
 //alert - https://github.com/twbs/bootstrap/blob/v4-dev/scss/alert.scss
 .alert {
-  text-align: center;
-  color: theme-color('text');
+  text-align: var(--cx-text-align, center);
+  color: var(--cx-color, var(--cx-g-color-text));
   @include type('6', 'semi');
-  padding: 22px 4rem 22px 20px;
-  line-height: 1.5;
+  padding: var(--cx-padding, 22px 4rem 22px 20px);
+  line-height: var(--cx-line-height, 1.5);
 
   &-icon {
-    height: 26px;
-    margin-right: spacer(2);
-    position: relative;
-    display: inline-block;
+    height: var(--cx-height, 26px);
+    margin-right: var(--cx-margin-right, spacer(2));
+    position: var(--cx-position, relative);
+    display: var(--cx-display, inline-block);
+    vertical-align: var(--cx-vertical-align, middle);
```

Comment:
Using `top` instead of `bottom` seems to give better results:
<img width="438" alt="Screen Shot 2019-05-15 at 20 41 34" src="https://user-images.githubusercontent.com/11561723/57818633-b3dadf00-7752-11e9-85b3-aa1b1a5588d7.png">

Reply:
I updated the branch and triggered the build